config_models/tests/test_decorators.py

```diff
@@ -2,12 +2,12 @@
 Tests for config models decorators
 """
 
-from __future__ import unicode_literals, absolute_import
+from __future__ import absolute_import, unicode_literals
 
 from config_models import decorators
 from config_models.models import ConfigurationModel
 from config_models.tests.utils import CacheIsolationTestCase
-from django.utils import six
+from django.utils import six  # pylint: disable=wrong-import-order
```

Comment:
You could also fix the warning by moving this line above the `from config_models...` imports; 3rd party imports like Django normally go above imports from the current package.